yoti-sdk-api/src/main/java/com/yoti/api/client/ActivityFailureException.java

```diff
@@ -13,6 +13,7 @@ public class ActivityFailureException extends ProfileException {
     private static final long serialVersionUID = 4496989849870743641L;
 
     private static final String ERROR_MSG = "Sharing activity unsuccessful for %s%s";
+    private static final String ERROR_DETAILS = " - Error[%n\tcode='%s' %n\tdescription='%s' %n\treason='%s'%n]";
```

Comment:
But the ERROR_DETAILS is most of the times not available so it would leave a msg like

```
Sharing activity unsuccessful for 'Something went wrong with this and that -'